### PR TITLE
resolves #25 regarding requirements of Technical Charter

### LINKED ISSUE #25: Update CONTRIBUTING.md to include list of committers per LF technical charter
The technical charter requires the CONTRIBUTING.md file in the repository to contain a list of TSC members/committers, or to document an alternative method of designation.

Re: https://apptainer.org/technical-charter section 2(b)

### PR DESCRIPTION
**Pull request by PaulCharlton**
update CONTRIBUTORS.md and CONTRIBUTING.md per Technical Charter requirements

update globally email address(es) for Greg Kurtzer

### CODE REVIEW

**Review by DrDaveD — APPROVED** [review]
Ok with me, but let's see if @gmkurtzer  approves of the email addresses for him.

File: .mailmap [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Gregory M. Kurtzer <g@sylabs.io>
+Gregory M. Kurtzer <gmk@resf.org>, <gmkurtzer@gmail.com>, <gmk@ctrliq.com>
```
Comment (gmkurtzer) [inline .mailmap:1]:
```suggestion
Gregory M. Kurtzer <gmkurtzer@gmail.com>
```

**Review by gmkurtzer — CHANGES_REQUESTED** [review]
Approved with the email address fix...

Wow, I had no idea my email was still referencing sylabs in places. ouch!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ install, use, modify, prepare derivative works, incorporate into other computer
...
+
+### Technical Steering Committee
+
+(in alphabetical sort order of surname)
+
+```text
+- Cedric Clerget <cedric@ctrliq.com>, <cedric.clerget@univ-fcomte.fr>
+- Dave Dykstra <dwd@fnal.gov>
+- Ian Kaneshiro <ian@ctrliq.com>, <iankane@umich.edu>
+- Gregory M. Kurtzer <gmk@resf.org>, <gmkurtzer@gmail.com>, <gmk@ctrliq.com>
```
Comment (gmkurtzer) [inline CONTRIBUTING.md:28]:
```suggestion
- Gregory M. Kurtzer <gmkurtzer@gmail.com>
```

File: .mailmap (hunk 1 above)
Comment (PaulCharlton) [reply .mailmap]: reduced to gmkurtzer@gmail.com

File: CONTRIBUTING.md (hunk 2 above)
Comment (PaulCharlton) [reply CONTRIBUTING.md]: reduced to gmkurtzer@gmail.com

**Comment by PaulCharlton** [comment]
@DrDaveD  @gmkurtzer the requested changes were addressed and GitHub is not counting due to some issue GitHub has with squashed merges

**Review by DrDaveD — APPROVED** [review]

**Pull request merged**